jar/src/clj/sixsq/slipstream/client/api/lib/run.clj

```diff
@@ -30,83 +30,60 @@
    * `webapp.1` is the name of the instance `1` of the application component `webapp`.
 
 
-  # Usage and examples
+  ## Usage and examples
 
-  ```
-  (require '[com.sixsq.slipstream.clj-client.lib.run :as r])
+  Use [[login]] and/or [[login-global]] to get token or alter API global authenticantion
+  context.
 
-  (def context {:username \"foo\"
-                :password \"bar\"})
-  (r/with-new-context context (r/get-run-info \"123-456\"))
+      (require '[sixsq.slipstream.clj-client.api.authn :as a])
+      (require '[sixsq.slipstream.clj-client.api.lib.run :as r])
```

Comment:
Are these namespaces correct? I think that I renamed `clj-client` everywhere to just `client`.